### ISSUE TITLE
About the training time.

### ISSUE THREAD

**Issue by livc** [tatsuokun/deepdep#1]
Hi, thanks for the awesome implementation first!

You said 'training time is approximately 9 minutes for 5 iterations' in the README, but the training cost me about 45min in an NVIDIA P40 GPU with the default setting in the repo. So I just want to confirm that it's 9min for an epoch or 5 epoch in your experiment?

Thanks.

**Comment by tatsuokun** [comment]
Hi livc, thank you for dropping by this repo and I appreciate your comments.

In short, you would take 30mins for training with ELMo.
Without ELMo, the expected time would be changed to 10mins.

Here's the result I got from a GPU server that has 1080Ti by running the code with 5 iters.
```
real    30m27.455s
user    30m32.164s
sys     0m39.248s
```
The training time is no longer 9 mins, instead 30 mins.
In the default setting, the code runs with ELMo that takes a few mins for its computation.
If we run the code without ELMo, the expected time would be
```
real    11m35.499s
user    11m17.276s
sys     0m18.960s
```

I realized that the README is outdated. So I'll modify the README, thank you!
